teuthology/task/internal.py

```diff
@@ -342,6 +342,50 @@ def push():
     except Exception:
         log.exception("Error pushing inventory")
 
+BUILDPACKAGES_SWAPPED = 0
+BUILDPACKAGES_OK = 1
+BUILDPACKAGES_REMOVED = 2
+BUILDPACKAGES_NOTHING = 3
+
+def buildpackages(ctx, config):
```

Comment:
Curious... why isn't this implemented as a class-based ``Task`` with https://github.com/ceph/teuthology/blob/master/teuthology/task/__init__.py#L9 ?